src/preprocess.py

```diff
@@ -351,50 +351,50 @@ def build_tasks(args):
     log.info("\tFinished indexing tasks")
 
     # 5) Initialize tasks with data iterators.
-    assert not (args.training_data_fraction < 1 and args.eval_data_fraction < 1), \
-        "training_data_fraction and eval_data_fraction could not be used at a same time (could not be < 1 together)"
+    assert not (args.pretrain_data_fraction < 1 and args.target_train_data_fraction < 1), \
```

Comment:
why is this assert a thing? It seems like these should be independent. Maybe a question for @sleepinyourhat

Reply:
Yeah I agree actually

Reply:
Another issue to make! 
I think we should have a separate PR for this, because this will change logging/ the way we're logging our metrics, and will require further testing.

Reply:
I vaguely remember that this happens during data loading, and that they both use the same mechanism. Should be an easy-ish fix, but I'm not sure why we need to support this.